### PR TITLE
Make sure all logs in production has JSON format

### PR DESCRIPTION
**Pull request by tomas-stefano**
## Context

Our logs are not well formatted in Logit.

## Changes proposed in this pull request

Make sure logs has the JSON format.

## Know issue

There is an issue with some of the logs when requests kicked in:

```
14:11:53 web.1    | - -> /ping
```

This is because of WEBrick logs. Soon we will be replacing WEBrick so I didn't solve this particular issue.

## Guidance to review

1. Does it log in JSON in the web app?
2. Does it log in JSON in the worker app?

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -68,5 +68,12 @@ class Application < Rails::Application
 
     config.exceptions_app = routes
     config.active_job.queue_adapter = :sidekiq
+
+    config.rails_semantic_logger.format = :json
+    config.semantic_logger.add_appender(
+      io: $stdout,
+      level: config.log_level,
+      formatter: config.rails_semantic_logger.format,
+    )
```
Comment (tomas-stefano) [inline config/application.rb:77]: This one is only to test the review app since the review apps are not `Rails.env.production?`.

Comment (tomas-stefano) [reply config/application.rb]: Ignore. Review environment requires the production config.

**Comment by tomas-stefano** [comment]
[Fix logs in LOGIT for Publish](https://trello.com/c/DclrioND/9-fix-logs-in-logit-for-publish)

File: app/lib/custom_log_formatter.rb [hunk 2]
```diff
@@ -57,14 +57,3 @@ def method_is_post_or_put_or_patch?
     hash.dig(:payload, :method).in?(%w[PUT POST PATCH])
   end
 end
-
-unless Rails.env.local?
-  Rails.application.configure do
-    config.semantic_logger.application = Settings.application_name
```
Comment (tomas-stefano) [inline app/lib/custom_log_formatter.rb:63]: Removed this line because `Settings.application_name` is nil in all environments.

**Review by Nitemaeric — APPROVED** [review]

**Pull request merged**